resources/views/twill/exhibitions/form.blade.php

```diff
@@ -10,217 +10,216 @@
-            'label' => 'Sponsors',
-            'note' => 'Display content blocks from this sponsor',
-            'max' => 1
-        ])
-    </a17-fieldset>
+    <x-twill::formFieldset id="sponsors" title="Sponsors">
+        <x-twill::browser
+            name='sponsors'
+            label='Sponsors'
+            route-prefix='exhibitionsEvents'
+            module-name='sponsors'
+            note='Display content blocks from this sponsor'
+            :max='1'
+        />
+    </x-twill::formFieldset>
 
-    <a17-fieldset id="waitTime" title="Wait time">
-        @formField('browser', [
-            'routePrefix' => 'exhibitionsEvents',
-            'moduleName' => 'waitTimes',
-            'name' => 'waitTimes',
-            'label' => 'Wait Time',
-            'note' => 'Select a queue to display the wait time for',
-            'max' => 1
-        ])
+    <x-twill::formFieldset id="waitTime" title="Wait time">
+        <x-twill::browser
+            name='waitTimes'
+            label='Wait Time'
+            note='Select a queue to display the wait time for'
+            route-prefix='exhibitionsEvents'
+            module-name='waitTimes'
+            :max='1'
+        />
 
-        @formField('wysiwyg', [
-            'name' => 'wait_time_override',
-            'label' => 'Wait time copy',
-            'maxlength' => 255,
-            'note' => 'Content will display below wait time data',
-            'toolbarOptions' => [
-                'italic'
-            ],
-        ])
-    </a17-fieldset>
+        <x-twill::wysiwyg
+            name='wait_time_override'
+            label='Wait time copy'
+            note='Content will display below wait time data'
+            :maxlength='255'
+            :toolbar-options="[ 'italic' ]"
+        />
+    </x-twill::formFieldset>
 
-    <a17-fieldset id="attributes" title="Attributes">
-        @formField('input', [
-            'name' => 'datahub_id',
-            'label' => 'Datahub ID',
-            'disabled' => true
-        ])
+    <x-twill::formFieldset id="attributes" title="Attributes">
+        <x-twill::input
+            name='datahub_id'
+            label='Datahub ID'
+            disabled='true'
+        />
 
-        @formField('multi_select', [
-            'name' => 'siteTags',
-            'label' => 'Tags',
-            'options' => $siteTagsList,
-            'placeholder' => 'Select some tags',
-        ])
-    </a17-fieldset>
-    <a17-fieldset id="related" title="Related">
-        @formField('browser', [
-            'routePrefix' => 'exhibitionsEvents',
-            'max' => 4,
-            'name' => 'exhibitions',
-            'label' => 'Related exhibitions'
-        ])
+        <x-twill::multi-select
+            name='siteTags'
+            label='Tags'
+            placeholder='Select some tags'
+            :options='$siteTagsList'
+        />
+    </x-twill::formFieldset>
+    <x-twill::formFieldset id="related" title="Related">
+        <x-twill::browser
```

Comment:
On my local machine, when I go to the [exhibitions edit page](http://admin.www-dev.artic.edu/exhibitionsEvents/exhibitions/874/edit), I'm getting an error at this field:
![Screenshot 2025-01-27 at 1 34 02 PM](https://github.com/user-attachments/assets/fb18b811-71a2-476f-911a-6967adcca8ba)